xml/security_ldap_sssd.xml

```diff
@@ -197,4 +197,23 @@ session:</screen>
     See <xref linkend="cha-security-auth"/> for information on managing the
     sssd.service with systemctl.
   </para>
+
+  <sect2 xml:id="sec-security-ldap-server-sssd-unsupported-password-hash">
+   <title>Unsupported passworded hashes and authentication schemes</title>
+   <para>
+    The following password hash and authentication schemes are not officially supported on &ds389;:
```

Comment:
The following are not supported as configuration values in dse.ldif for the settings nsslapd-rootpwstoragescheme or passwordStorageScheme, or as a value of passwordStorageScheme in account policy objects. Note that database imports that contain these values *are* supported provided nsslapd-enable-upgrade-hash is `on` (defaults to `on`). 

* SHA
* SSHA
* SHA256
* SSHA256
* SHA384
* SSHA384
* SHA512
* SSHA512
* NS-MTA-MD5
* clear
* MD5
* SMD5